horace_core/sqw/@rundatah/private/calc_sqw_.m

```diff
@@ -20,10 +20,10 @@
 %
 % Output:
 % --------
-%   w               Output sqw object
-%   grid_size       Actual size of grid used (size is unity along dimensions
+%   w              Output sqw object
+%   grid_size      Actual size of grid used (size is unity along dimensions
 %                  where there is zero range of the data points)
-%   urange          Actual range of grid - the specified range if it was given,
+%   pix_range      Actual range of grid - the specified range if it was given,
```

Comment:
the return value is `img_range` not `pix_range`

Reply:
from one side, it goes to axis so it is img_range. From other side, pixels are sorted and constrained according to this value, so its pix_range. What it is actually, is the subj of #447. Changed here to be consistent to pix_range